### PR TITLE
Create idleleo.com.xml

### PR DESCRIPTION
**Pull request by paniy**

### CODE REVIEW

File: src/chrome/content/rules/idleleo.com.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="idleleo">
+	<target host="d.idleleo.com" />
+	<target host="i.idleleo.com" />
+	<target host="idleleo.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/idleleo.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586022926#L325

^ to www

File: src/chrome/content/rules/idleleo.com.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<ruleset name="idleleo">
+	<target host="d.idleleo.com" />
```
Comment (ivysrono) [inline src/chrome/content/rules/idleleo.com.xml]: https://travis-ci.org/EFForg/https-everywhere/jobs/586022926#L325

Move to comments.

**Comment by ivysrono** [comment]
@zoracon 
Why the preload test is always error on this pr?
https://travis-ci.org/EFForg/https-everywhere/jobs/586882247

**Comment by paniy** [comment]
> @zoracon
> Why the preload test is always error on this pr?
> https://travis-ci.org/EFForg/https-everywhere/jobs/586882247
> `No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`

Maybe the firewall of this Web block it?

**Comment by ivysrono** [comment]
> 
> 
> > @zoracon
> > Why the preload test is always error on this pr?
> > https://travis-ci.org/EFForg/https-everywhere/jobs/586882247
> > `No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself.`
> 
> Maybe the firewall of this Web block it?

I don't thinks so. Preload test wether the target is in HSTS Proload List that no need by this extension.

**Comment by ivysrono** [comment]
Status: idleleo.com is currently preloaded.
https://hstspreload.org/?domain=idleleo.com

**Pull request closed without merging**